src/main/scala/it/pagopa/interop/notifier/service/impl/EventIdRetriever.scala

```diff
@@ -17,7 +17,7 @@ import it.pagopa.interop.commons.logging.{CanLogContextFields, ContextFieldsToLo
 import java.util.UUID
 import scala.concurrent.{ExecutionContext, Future}
 
-final class EventIdRetriever(
```

Comment:
same, any reason to remove `final`

Reply:
No `final`  is appropriate here but look at this [comment](https://github.com/pagopa/interop-be-notifier/pull/22/files#r979737431) anyway.